projects/core/src/configurator/commons/store/actions/configurator-group.actions.ts

```diff
@@ -0,0 +1,2 @@
+export * from './configurator-cart.action';
+export * from './configurator.action';
```

Comment:
I find the name of the file a bit misleading, I thought it contains actions for the groups.
Why not just expose it in the index?

Reply:
'group' means here all actions for the group of configurator actions, similar like cart-group.actions.ts